recipes/sahi/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sahi" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sahi-{{ version }}.tar.gz
+  sha256: 4b6855a1db49988c775759cf9ccafa875c03267e1ef6856f84094cc9e5728452
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - fire
+    - opencv >=4.2.0.32
+    - pillow >=8.2.0
+    - python >=3.7
+    - pyyaml
+    - shapely >=1.7.0
+    - tqdm >=4.48.2
+
+test:
+  imports:
+    - sahi
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/obss/sahi
+  summary: A vision library for performing sliced inference on large images/small objects
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - fcakyon
```

Comment:
Cc @fcakyon, just need to confirm that you're happy with being a recipe maintainer here (need an explicit yes/no). Most of the work is automated so that a PyPI release triggers an auto-build. So it usually just involves clicking a button to merge in a PR when new releases come out (plus a few updates to the dependency pins if necessary).

Reply:
So when sahi makes a release on PyPI, there will be a conda-forge bot that opens a PR automatically. We'll get a ping about that conda release PR, and can check it before merging.

Reply:
No... There's something about the requirements.txt missing. Lemme try and debug this.

Reply:
So the `requirement.txt` file needs to be included in the `sahi` sdist, I've opened up a PR at https://github.com/obss/sahi/pull/217 to add it. Once that PR is merged, you'll need to do a sahi v0.8.1 or v0.9.0 release, and then I'll continue with the conda-forge packaging here.